docs/src/components/ComponentDoc/ComponentDocTag.tsx

```diff
@@ -1,27 +1,84 @@
 import * as React from 'react'
 import * as _ from 'lodash'
 import { Header, Message } from 'semantic-ui-react'
+const behaviorMenuItems = require('docs/src/behaviorMenu')
 
 const headerStyle = {
   whiteSpace: 'pre-line',
 }
 
+type behaviorFiltered = {
+  type: string
+  displayName: string
+  defaultFile: string
+}
+
 class ComponentDocTag extends React.Component<any, any> {
   getTagDescription = (forTag, fromInfo) => {
     const tags = (fromInfo.docblock && fromInfo.docblock.tags) || []
     return _.result(_.find(tags, 'title', forTag), 'description')
   }
 
+  getBehaviorURLandName = fromInfo => {
+    const defaultBeharior = {
+      name: undefined,
+      url: undefined,
+    }
+    const accProperty = fromInfo.props.find(property => property.name === 'accessibility')
+    if (!accProperty) {
+      return defaultBeharior
+    }
+    const accPropertyFormatted = accProperty.defaultValue.split('.').pop() + '.ts'
+    const behaviorFiltered: behaviorFiltered[] = []
+    behaviorMenuItems.forEach(behavior => {
+      return behavior.variations.find(variation => {
+        if (variation.name === accPropertyFormatted) {
+          behaviorFiltered.push({
+            type: behavior.type,
+            displayName: behavior.displayName,
+            defaultFile: variation.name,
+          })
+        }
+      })
+    })
+
+    if (behaviorFiltered && behaviorFiltered.length > 0) {
+      const behaviorsFolder = behaviorFiltered[0].type + 's'
+      const behaviorFile = behaviorFiltered[0].defaultFile.replace('.ts', '')
+      return {
+        name: `${behaviorFile}`,
+        url: `${behaviorsFolder}/${behaviorFiltered[0].displayName}#${_.kebabCase(behaviorFile)}`,
+      }
+    }
+    return defaultBeharior
+  }
+
+  getDefaultBehavior(info) {
+    const defaultBehavior = this.getBehaviorURLandName(info)
+    if (defaultBehavior && defaultBehavior.name && defaultBehavior.url) {
+      return (
+        <span>
+          Default behavior: <a href={defaultBehavior.url}> {defaultBehavior.name} </a>{' '}
+        </span>
+      )
+    }
+  }
+
   render() {
     const { info, tag, title, errorMessage } = this.props
-    const description = this.getTagDescription(tag, info) || (
-      <Message error content={errorMessage} compact={true} />
-    )
+    const accDescription = this.getTagDescription(tag, info)
+    const description =
+      accDescription || this.getBehaviorURLandName(info).name ? (
```

Comment:
You are calling `getBehaviorURLandName()` twice in `render()` (The second call is in `getDefaultBehavior()`)